tests/test_op6_schema_validation.py

```
@@ -858,5 +858,195 @@ def test_start(self):
+        # Validate instance — should fail because base is abstract
+        Step(
+            RunRequest("validate instance of abstract type should fail")
+            .post("/validate-instance")
+            .with_json({
+                "instance_id": "gts.x.test6.abstract.base.v1~x.test6._.my_item.v1",
+            })
+            .validate()
+            .assert_equal("status_code", 200)
+            .assert_equal("body.ok", False)
+        ),
+    ]
```

Ruff (0.15.7)

[warning] Mutable default value for class attribute

(RUF012)